cx_Freeze/command/bdist_msi.py

```diff
@@ -117,6 +117,7 @@ class bdist_msi(Command):
         ("target-name=", None, "name of the file to create"),
         ("target-version=", None, "version of the file to create"),
         ("upgrade-code=", None, "upgrade code to use"),
+        ("license-file=", None, "license file to include in the installer"),
```

Comment:
Does it only accept .rtf and .txt? It is advisable to include the accepted formats.

Reply:
Only RTF I’ve gotten to render so far

Reply:
[`e3c500a` (#2472)](https://github.com/marcelotduarte/cx_Freeze/pull/2472/commits/e3c500a76d064fcf5d2d17b58200e2964c6affe0)

Reply:
RTF will be the only supported file. See https://learn.microsoft.com/en-us/windows/win32/msi/scrollabletext-control#control-attributes -> Text for more details

Reply:
Please add this information to the docs, like in the https://cx-freeze--2472.org.readthedocs.build/en/2472/bdist_msi.html#cmdoption-arg-upgrade_code that has a link.
Also fix the option https://cx-freeze--2472.org.readthedocs.build/en/2472/bdist_msi.html#cmdoption-arg-license-file to license_file.